source/main.cpp

```diff
@@ -1,10 +1,26 @@
 #include <string>
 
 #include "PasswordManager.h"
+#include "DataBase/DataBaseHandler.hpp"
+#include "DataBase/common/DataBaseCommon.hpp"
 
-int main() {
+int main()
+{
     std::string password = "haslo";
     PasswordManager passwordManager(password);
+    const std::vector<std::string> passwordValues{
+        "1", "test", "test", "test", "test", "test", "test"};
+    try
+    {
+        dataBase::DataBaseHandler dataBaseHandler(dataBase::common::dataBasePath);
+        dataBaseHandler.insertData(dataBase::common::passwords, dataBase::common::passwordRecords, passwordValues);
+        std::cout << "dataBase is working" << std::endl;
```

Comment:
this is suitable for a unit test :)

Reply:
Yes yes. I cover uts during DataBaseManagmentTask :)